pkg/daemon/multus/nginx-pod.yaml

```diff
@@ -0,0 +1,52 @@
+# This is a Pod and not a Deployment because there is intended to only be one running
+apiVersion: apps/v1
+kind: Pod
+metadata:
+  name: multus-validation-test-web-server
+  labels:
+    app: multus-validation-test-web-server
+    app.kubernetes.io/name: "nginx"
+    app.kubernetes.io/instance: "nginx" # there is only one instance
+    app.kubernetes.io/component: "server"
+    app.kubernetes.io/part-of: "multus-validation-test"
+    app.kubernetes.io/managed-by: "kubectl-rook-ceph"
+  annotations:
+    k8s.v1.cni.cncf.io/networks: "{{ .NetworksAnnotationValue }}"
+spec:
+  # TODO: selectors, affinities, tolerations
```

Comment:
Currently the tool will only test nodes where there are no taints?

Reply:
Right. This was a fine assumption on a brand new OpenShift cluster, but it is a good future to-do that should also be easy low-hanging fruit.